pkg/core/block/block_test.go

```diff
@@ -104,6 +104,7 @@ func TestHashBlockEqualsHashHeader(t *testing.T) {
 	assert.Equal(t, block.Hash(), block.Header.Hash())
 }
 
+/*
```

Comment:
Create an issue for these binary test-cases?

Reply:
Take something from C# privnet?

Reply:
Fixed.